### DISCUSSION TITLE
debug logfile location

### DISCUSSION THREAD

**Discussion by damnms** [CastagnaIT/plugin.video.netflix#1115] [General]
i can not watch any movie on netflix with that addon since a couple of weeks.
in my browser (chrome on linux) it works like a charm.
so i enabled kodis debug logging and the netflix addon logging but dont know where to find that debug log for the addon
i checked /storage/.kodi/userdata/addon_data/plugin.video.netflix but there is (imo) nothing related
in the kodi.log i also see nothing related to that behaviour, just tons of messages about the youtube plugin

i am running coreelec stable (kodi 19.1).
the video seems to start but then nothing happens, only the play information in the upper right corner is visible.
after that this info stays and i can not play anything (no music, no tv stream etc.)

**Comment by amyren1966** [comment]
Did you by any chance update widevine cdm just before it stopped working? (I did start another thread with that issue)

**Comment by damnms** [comment]
no, i switched the codec in settings to something incompatible and therefore it was not working.
